lib/embulk/output/bigquery/bigquery_client.rb

```diff
@@ -440,6 +440,12 @@ def create_table_if_not_exists(table, dataset: nil, options: nil)
               }
             end
 
+            options['expiration_time'] ||= nil
```

Comment:
IMO, this is not necessary. If expration_time is nil or false, assign nil.

Reply:
Thx! I remove the line 👍